change/@fluentui-react-button-159dc420-be08-4472-9dbb-634db2fd56ca.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Button: Changing border radius of small button to borderRadiusMedium.",
```

Comment:
Nit: for this type of change I always find a comment like:

> Button: Changing border radius of small button to borderRadiusMedium from borderRadiusSmall

to be really nice.

Reply:
I'll make the change!